PigLatin/PigLatin.cs

```diff
@@ -1,21 +1,86 @@
 ﻿using System;
+using System.Linq;
 
 namespace PigLatin
 {
-    class Program
+  class Program
+  {
+    public static void Main()
     {
-        public static void Main()
-        {
-            // your code goes here
+      // your code goes here
+      Console.WriteLine("Enter word: ");
+      String userInput = Console.ReadLine();
+
+      String firstLetter = userInput.Substring(0, 1);
+      String restOfWord = userInput.Substring(1);
+
+
+      char[] vowels = new char[] { 'a', 'A', 'e', 'E', 'i', 'I', 'o', 'O', 'u', 'U' };
+      char[] otherLetters = {'b','B','c','C','d','D','f','F','g','G','h','H','j','J','k','K','l','L','m','M','n','N','p','P',
+                                    'q','Q','r','R','s','S','t','T','u','U','v','V','x','X','y','Y','z','Z'};
```

Comment:
You have a little bug in your otherLetters array. you have U in there but i think you meant to have W.